src/Select/SelectInput.js

```diff
@@ -53,7 +53,7 @@ export type Props = {
    * @param {object} event The event source of the callback
    * @param {object} child The react element that was selected
    */
-  onChange?: (event: Object, child: Element<*>) => void,
+  onChange?: (event: SyntheticMouseEvent<*>, child: Element<*>) => void,
```

Comment:
I believe this should probably be the more super `SyntheticUIEvent<>` (parent class to `SyntheticMouseEvent`)